src/connectors/infinite-hits/connectInfiniteHits.ts

```diff
@@ -12,16 +12,52 @@ import {
   addQueryID,
   noop,
 } from '../../lib/utils';
-import { InfiniteHitsRendererWidgetParams } from '../../widgets/infinite-hits/infinite-hits';
 
-export type InfiniteHitsConnectorParams = Partial<
-  InfiniteHitsRendererWidgetParams
->;
+export type InfiniteHitsConnectorParams = {
+  /**
+   * Escapes HTML entities from hits string values.
+   *
+   * @default `true`
+   */
+  escapeHTML?: boolean;
+
+  /**
+   * Enable the button to load previous results.
+   *
+   * @default `false`
+   */
+  showPrevious?: boolean;
+
+  /**
+   * Receives the items, and is called before displaying them.
+   * Useful for mapping over the items to transform, and remove or reorder them.
+   */
+  transformItems?: TransformItems<
+    AlgoliaHit & {
```

Comment:
Is this `AlgoliaHit` or `Hit`?  The `Hit` also contains `__position`.

Reply:
it's Hit, and it doesn't have __hitIndex I think: https://github.com/algolia/instantsearch.js/blob/d5f21a4e9a0230e56f133347f57acd7f09bb34f8/src/lib/utils/hits-absolute-position.ts#L10

Reply:
We call `transformItems` after all the transformations, so this is with the internal hit containing the position and `queryId`:

https://github.com/algolia/instantsearch.js/blob/d8f6818a31d195ab7d818f5fdb5661256741a1b5/src/connectors/infinite-hits/connectInfiniteHits.ts#L192